src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -492,6 +408,22 @@ def create_bcache(devicegraph, bcache, backing_devname, caching_devname)
         bcache_creator.create_bcache(new_bcache, backing_devname, caching_devname)
       end
 
+      # Creates a volume group in the given devicegraph
+      #
+      # @param devicegraph [Devicegraph]                    Starting devicegraph
+      # @param vg          [Planned::LvmVg]                 Volume group
+      # @param pvs         [Planned::Partition,Planned::Md] List of physical volumes
+      # @return            [Proposal::CreatorResult] Result containing the specified volume group
+      def create_logical_volumes(devicegraph, vg, pvs)
+        lvm_creator = Proposal::LvmCreator.new(devicegraph)
+        lvm_creator.create_volumes(vg, pvs)
+      rescue RuntimeError
```

Comment:
that a bit scary me that it just catch any runtime error and do not even log it. What if something else happens then what you expect? I think we should log at least exception message.

Reply:
Again, this was not added in this PR. I will log the error message.